src/Sulu/Bundle/AdminBundle/Resources/js/containers/Form/types.js

```diff
@@ -121,12 +121,17 @@ export interface FormStoreInterface {
     +options: SchemaOptions,
     +resourceKey: ?string,
     +schema: Object,
+    +set: (name: string, value: mixed) => void,
```

Comment:
we should get rid of the `FormStoreInterface::set` and `FormStoreInterface::setMultiple` methods and use `change` and `changeMultiple` with a context instead

Reply:
parameters of these methods should use `dataPath` instead of `name`